selfie.c

```diff
@@ -5594,8 +5594,9 @@ void store_data(uint64_t baddr, uint64_t data) {
 void emit_instruction(uint64_t instruction) {
   store_instruction(binary_length, instruction);
 
-  if (*(code_line_number + binary_length / INSTRUCTIONSIZE) == 0)
-    *(code_line_number + binary_length / INSTRUCTIONSIZE) = line_number;
+  if (code_line_number != (uint64_t*) 0)
+    if (*(code_line_number + binary_length / INSTRUCTIONSIZE) == 0)
+      *(code_line_number + binary_length / INSTRUCTIONSIZE) = line_number;
```

Comment:
These changes were introduced by Thomas Wulz before we did the branch split (commit 120e2b71).

We made no changes whatsoever to Selfie. Selfie as currently in the `master` branch will work perfectly fine